src/components/SearchResultItem.tsx

```diff
@@ -22,12 +23,23 @@ export class SearchResultItem extends React.Component<SearchResultItemProps, Sea
         let highLightedItemText = getHighlightedText(this.props.data.text, this.props.highlightedText, true);
         let highLightedCategoryText = getHighlightedText(this.props.category, this.props.highlightedText, false);
         let ItemTypeIconPath = "src/resources/icons/library-" + this.props.data.itemType + ".svg";
+        let ItemDescription: JSX.Element = null;
+
+        if (this.props.detailed) {
+            let description = "No description available";
```

Comment:
Is this the current behaviour for Dynamo? I'm thinking `ItemDescription` can remain `null` if there's no description (to simplify the DOM tree), does that make sense? Anyway, both `ItemDescription` and `ItemTypeIconPath` should be camel case.